modules/storage-ephemeral-vols-procedure.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * storage/generic-ephemeral-vols.adoc
+
+:_content-type: PROCEDURE
+[id="generic-ephemeral-vols-procedure_{context}"]
+= Creating generic ephemeral volumes
+
+.Procedure
+
+. Create the `pod` object definition and save it to a file.
+
+. Include the generic ephemeral volume information in the file.
++
+.my-example-pod-with-generic-vols.yaml
+[source, yaml]
+----
+kind: Pod
+apiVersion: v1
+metadata:
+  name: my-app
+spec:
+  containers:
+    - name: my-frontend
+      image: busybox:1.28
+      volumeMounts:
+      - mountPath: "/mnt/storage"
+        name: data
+      command: [ "sleep", "1000000" ]
+  volumes: <1>
+    - name: data
```

Comment:
```suggestion
  volumes: 
    - name: data <1>
```
Maybe better. As accually the volumes is a list, and Generic ephemeral volume is the volumes[0].

Reply:
Done. Thanks!